src/state/DB.js

```diff
@@ -167,6 +167,14 @@ class DB {
         );
     }
 
+	 getMainSheet(info) {
+			for (var sheetIndex = 0; sheetIndex < info.worksheets.length; sheetIndex){
+				if(!this.config.sheets.worksheet_name) break;
```

Comment:
can you add the `worksheet_name` property in https://github.com/Team4159/CardinalHours/blob/3644172bbe7ed649be1ef8d1fd532184490bbf7f/src/state/default_config.json#L1-L12
and remove the `worksheet_id` property there.

Reply:
Should I keep id as a backup?

Reply:
Scratch that, It dies.

Reply:
Should I keep it in the code for outdated?